gax-java/gax/src/test/java/com/google/api/gax/rpc/EndpointContextTest.java

```diff
@@ -40,14 +40,22 @@
 import java.io.IOException;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
 import org.mockito.Mockito;
+import uk.org.webcompere.systemstubs.environment.EnvironmentVariables;
+import uk.org.webcompere.systemstubs.jupiter.SystemStub;
+import uk.org.webcompere.systemstubs.jupiter.SystemStubsExtension;
 
+@ExtendWith(SystemStubsExtension.class)
 class EndpointContextTest {
+
   private static final String DEFAULT_ENDPOINT = "test.googleapis.com:443";
   private static final String DEFAULT_MTLS_ENDPOINT = "test.mtls.googleapis.com:443";
   private EndpointContext.Builder defaultEndpointContextBuilder;
   private StatusCode statusCode;
 
+  @SystemStub private final EnvironmentVariables environmentVariables = new EnvironmentVariables();
```

Comment:
Looks like there's no straightforward way to set environment variables in Junit 5.

I followed this [post](https://stackoverflow.com/questions/46846503/properly-set-system-properties-in-junit-5/59635733#59635733) to use systemstubs.